packages/@glimmer/application/src/loaders/bytecode/resolver.ts

```diff
@@ -18,6 +18,10 @@ function buildComponentDefinition(ComponentClass: Factory<Opaque>, manager: Comp
   };
 }
 
+export interface TemplateMeta {
```

Comment:
I think this will work in tests but not in real apps. That's because the MU compiler delegate only includes `specifier` information, not the full `locator`. (It's wasteful to include module metadata in the payload that never gets used at runtime.)

Is it possible to change the template meta to just be `specifier: string`? I think that's all the compiler delegate emits.